src/openforms/authentication/urls.py

```diff
@@ -1,10 +1,19 @@
 from django.urls import path
 
-from .views import AuthenticationReturnView, AuthenticationStartView
+from .views import (
+    AuthenticationReturnView,
+    AuthenticationStartView,
+    RegistratorSubjectInfoView,
+)
 
 app_name = "authentication"
 
 urlpatterns = [
+    path(
+        "<slug:slug>/registrator-subject",
```

Comment:
I added this URL with the slug but in hindsight we don't actually use the form (maybe be should).

Reply:
let's leave it like this for consistency with the other URLs/endpoints - maybe in the future we'll need it or it'll be relevant input for a plugin.